core/src/test/resources/HistoryServerExpectations/blacklisting_for_stage_expectation.json

```diff
@@ -134,7 +135,8 @@
         "stderr" : "http://172.30.65.138:64278/logPage/?appId=app-20180109111548-0000&executorId=1&logType=stderr"
       },
       "schedulerDelay" : 10,
-      "gettingResultTime" : 0
+      "gettingResultTime" : 0,
+      "partitionId" : -1
```

Comment:
I see partitionId as -1 in the test cases everywhere, does this need some test case where it is >= 0?